docs/content/docs/faq.md

```diff
@@ -242,3 +242,21 @@ This also applies to Blazor, Xamarin, MAUI, and other mobile frameworks.
 ### What browsers are supported?
 
 {{% include file="_includes/browsers.md" %}}
+
+## Summarization
+
+### Summaries vs Snapshots
+
+#### Why do we need summaries and snapshots?
+Summaries capture the state of a container at a point in time. Snapshots allow new clients to quickly catch up to recent state. Without them, a client would have to apply every operation in the op log, even if those operations no longer affected the current state (e.g. op 1 inserts 'h' and op 2 deletes 'h'). For very large op logs, this would be very expensive both for the clients to process and to download from the service.
+Instead, when a client joins a collaborative document, they can download a snapshot of the container state, and simply process new operations from that point forward.
+
+#### Summary vs Snapshot
```

Comment:
Maybe I should name this section "Differences". Any thoughts?

Reply:
Also change to ###
Yea, we should change the Q to "What's the difference between summaries and snapshots?"

Reply:
Updated.